server/routes/loginStrategies/googleStrategy.js

```diff
@@ -27,20 +27,24 @@ passport.use(
                 let msg = userFromDb["msg"];
                 var user = null;
 
-            //if the message contains "success"
-            // (if user exists and was retrieved or new user and signinOption were created)
-            // extract the returned user information
-            if (msg.includes("success")){
-                user = userFromDb["user"];
-                logger.info(`[${user.userId}] Successfully signed in using ${user.serviceName}`);
-            } else {
-                //deal with error messages
-                logger.error(user.userId + ":" + msg);
+                // if the message contains "success"
+                // (if user exists and was retrieved or new user and signinOption were created)
+                // extract the returned user information
+                if (msg.includes("success")){
```

Comment:
nit: Format document 😄 
```suggestion
                if (msg.includes("success")) {
```